package.json

```diff
@@ -10,6 +10,14 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "publish": "npm publish --access public --tag next"
   },
+  "exports": {
+    ".": {
+      "require": "./dist/highlightjs-vue.min.js",
+      "import": "./dist/highlightjs-vue.esm.min.js",
+      "types": "./dist/vue.d.ts"
+    },
+    "./*": "./*"
```

Comment:
Why is this line necessary?

Reply:
[nodejs doc](https://nodejs.org/api/packages.html#package-entry-points): 

> As a last resort, package encapsulation can be disabled entirely by creating an export for the root of the package "./*": "./*". This exposes every file in the package at the cost of disabling the encapsulation and potential tooling benefits this provides. 

I'm not sure to understand why we need it, @Neophen can you elaborate ?

Reply:
Sorry that is not neccasary :D

Reply:
Let's remove i& it's not necessary.